pytorch_blade/tests/torchscript/since_1_14.graph

```diff
@@ -30,3 +30,17 @@ graph(%p1 : Float(*, *, *)):
 // CHECK: Float(*, *) = aten::var
   %1 : Tensor = aten::var(%p1, %dims, %cst0, %false)
   return (%1)
+
```

Comment:
Please add unittests for other functions. Add concrete shape analyses as well

Reply:
thanks, i added some ut in `since_1_14.graph`.